src/html_support_files/odoc.css

```diff
@@ -1253,7 +1274,7 @@ td.def-doc *:first-child {
   }
   .odoc-toc {
       position: unset;
-      max-height: unset;
+      max-height: unset !important;
```

Comment:
Maybe it would be better to have the same priority on both the rule setting the `max-height` and the one inside the media block?

Reply:
```css
/* When a search bar is present, we need the sticky sidebar to be a bit lower,
   so `top` is higher  */

.odoc-search + * + .odoc-tocs .odoc-toc {
  --toc-top: calc(var(--search-bar-height) + 2 * var(--search-padding-top));
  max-height: calc(100vh - 2 * var(--toc-top));
  top: var(--toc-top)
}
```
Here is it, the high priority is built in the rule.